docs/howtos/Client Generation/11etag.mdx

```diff
@@ -0,0 +1,64 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Conditional Request Header
+
+This doc details what emitters will generate for conditional request header
+
+## `If-Match` and `If-None-Match`
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+op get(
+  @header("If-Match")
+  @doc("The request should only proceed if an entity matches this string.")
+  ifMatch?: string,
+
+  @header("If-None-Match")
+  @doc("The request should only proceed if no entity matches this string.")
+  ifNoneMatch?: string,
+): void;
```

Comment:
`get` returning `void` seems weird...

Reply:
response in this case doesn't matter, but GET without response indeed be weird... Let me add a response type.